src/daemon/scheduler.rs

```diff
@@ -231,14 +240,38 @@ async fn try_sync_parent(
     }
 }
 
-fn make_cas_republish(caserver: Arc<CaServer>, actor: Actor) -> ScheduleHandle {
-    SkippingScheduler::run(120, "CA certificate republish", move || {
+fn make_cas_republish(caserver: Arc<CaServer>, event_queue: Arc<MessageQueue>) -> ScheduleHandle {
+    SkippingScheduler::run(
+        SCHEDULER_INTERVAL_SECONDS_REPUBLISH,
+        "CA certificate republish",
+        move || {
+            let mut rt = Runtime::new().unwrap();
+            rt.block_on(async {
+                debug!("Triggering background republication for all CAs, note this may be a no-op");
```

Comment:
Is this downgrade from info! to debug! worthy of mentioning in release notes I wonder?